### PR TITLE
Update package metadata

### PR DESCRIPTION
**Pull request by sebalix**
I don't know if putting the version number in `__init__.py` is a good practice or not? How to integrate the version given by the git tag?
cc @sbidoul

### CODE REVIEW

**Comment by sebalix** [comment]
Perhaps we can remove this `version` attribute from `setup.cfg` and `setuptools_scm` is doing all the magic there?

**Comment by sbidoul** [comment]
> Perhaps we can remove this version attribute from setup.cfg and setuptools_scm is doing all the magic there?

Yes, please don't put the version number in `__init__.py` it's much easier to release with setuptools_scm.

You can rebase on #6 (which I just merged) and run `pipx run build`: that should generate the distributions with some automatically computed version number.

**Comment by sbidoul** [comment]
BTW, if you want to make it modern and standards-based, you can drop `setup.cfg` and move the metadata to a `[project]` section in `pyproject.toml`. Recent versions of setuptools support that. See https://setuptools.pypa.io/en/latest/userguide/quickstart.html#basic-use and https://packaging.python.org/en/latest/specifications/declaring-project-metadata/.

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+author_email = sebastien.alix@camptocamp.com
+url = https://github.com/OCA/oca-port
+description = OCA tool to help with modules migration
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords = odoo, oca, port, migration, modules, addons
+license = LGPL-3.0
+classifiers =
+    License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)
+    Programming Language :: Python :: 3
```
Comment (simahawk) [inline setup.cfg:13]: we need some odoo classifiers too

Comment (sbidoul) [reply setup.cfg:13]: `Framework :: Odoo` exists

**Comment by sebalix** [comment]
@sbidoul @simahawk moved everything from `setup.cfg` to `pyproject.toml` file, I tried a build locally (`python3 -m build`) with a local tag, and things seems good in the archive created in `./dist/` folder.

File: pyproject.toml [hunk 2]
```diff
@@ -1,5 +1,38 @@
+[project]
+name = "oca_port"
+authors = [
+    {name = "Sébastien Alix", email="sebastien.alix@camptocamp.com"}
```
Comment (simahawk) [inline pyproject.toml:4]: Shall we have OCA too here as we do for modules?

**Review by simahawk — APPROVED** [review]

**Comment by sbidoul** [comment]
> @sbidoul @simahawk moved everything from `setup.cfg` to `pyproject.toml` file, I tried a build locally (`python3 -m build`) with a local tag, and things seems good in the archive created in `./dist/` folder.

Cool I'll publish a first version to the OCA account on PyPI and then add the token to GitHub secrets so the workflow will do it automatically afterwards.

File: pyproject.toml (hunk 2 above)
Comment (sebalix) [reply pyproject.toml:5]: Added

**Pull request merged**